test/unit/alphabet/alphabet_test.cpp

```diff
@@ -66,17 +85,18 @@ using alphabet_types = ::testing::Types<dna4, dna5, dna15, rna4, rna5, rna15,
                                         qualified<aa27, phred42>,
                                         qualified<gapped<dna4>, phred42>,
                                         gapped<qualified<dna4, phred42>>,
+                                        dna4q,
                                         gap,
                                         gapped<dna4>,
                                         gapped<dna15>,
                                         char, char16_t, // char32_t, too slow
                                         uint8_t, uint16_t, // uint32_t, too slow
-                                        dna4q,
```

Comment:
maybe remove this unnecessary change again. Or did this actually make a difference? ^^

Reply:
well dna4q is just another name for `qualified<dna4>` so it makes sense to group it together. But I assume this will change anyway if #535 is merged before this one.

Reply:
I thought this one is #535 :thinking: